### PR TITLE
Add Nano doc structure and Nano overview

### PR DESCRIPTION
**Pull request by yangw1234**
also add @zhentaocc

### CODE REVIEW

File: docs/readthedocs/source/doc/Nano/Overview/nano.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+```bash
+source bigdl-nano-init
+```
+
+The `bigdl-nano-init` scripts will export a few environment variable according to your hardware to maximize performance. 
+
+In a conda environment, this will also add this script to `$CONDA_PREFIX/etc/conda/activate.d/`, which will automaticly run when you activate your current environment.
+
+In a pure pip environment, you need to run `source bigdl-nano-init` every time you open a new shell to get optimal performance.
```
Comment (TheaperDeng) [inline docs/readthedocs/source/doc/Nano/Overview/nano.md:30]: Should we teach our users how to unset those variables (`source bigdl-nano-unset-env`)?

Comment (yangw1234) [reply docs/readthedocs/source/doc/Nano/Overview/nano.md:30]: good catch!

File: docs/readthedocs/source/doc/Nano/Overview/nano.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
+# Nano User Guide
```
Comment (jason-dai) [inline docs/readthedocs/source/doc/Nano/Overview/nano.md:1]: Organize the document as follows:

1. Overview
2. Install
3. Getting started
    - PyTorch: link to PyTorch Getting Started page (which provides a series examples/tutorials)
    - TensorFlow: link to TensorFlow Getting Started page (which provides a series examples/tutorials)

Comment (yangw1234) [reply docs/readthedocs/source/doc/Nano/Overview/nano.md:1]: sure, I'll reorganize the them.

**Comment by yangw1234** [comment]
@jason-dai 
How about this version:

Overview/nano.md:
1. Overview
2. Install
3. Getting started
    PyTorch: a very simple code snippet and a link to PyTorch Getting Started page 
    TensorFlow: a very simple code snippet and link to TensorFlow Getting Started page

QuickStart/pytorch.md: descriptions of features and links to notebooks
QuickStart/tensorflow.md: descriptions of features and links to notebooks

File: docs/readthedocs/source/doc/Nano/Overview/nano.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+---
+## **2. Install**
+
+BigDL-Nano can be installed using pip and we recommend installing BigDL-Nano in a conda environment.
+
+For PyTorch Users, you can install bigdl-nano along with some dependencies specific to PyTorch using the following command.
+
+```bash
+pip install bigdl-nano[pytorch]
```
Comment (jason-dai) [inline docs/readthedocs/source/doc/Nano/Overview/nano.md:15]:
```
conda create -n env
conda activate env
pip install bigdl-nano[pytorch]
```

Comment (yangw1234) [reply docs/readthedocs/source/doc/Nano/Overview/nano.md:17]: changed

**Comment by yangw1234** [comment]
http://10.112.231.51:18888/view/BigDL-2.0-PR/job/BigDL-NanoTests-PR-Validation/70/
http://10.112.231.51:18888/job/ZOO-PR-nano-pytorch-onnxruntime/60/

**Comment by jason-dai** [comment]
LGTM

**Comment by jason-dai** [comment]
Also need to add a user guide on how to install on Windows

**Comment by jason-dai** [comment]
Also need to add a user guide on how to install on Windows, and link it from the overview page @yangw1234

**Comment by yangw1234** [comment]
> Also need to add a user guide on how to install on Windows, and link it from the overview page @yangw1234

Ok, working on it.

**Pull request merged**